courses/machine_learning/intro_to_tensorflow/4_keras_functional_api.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>This is great! A couple of things:</p><p> </p><ul><li>Remove "Learn how to train a Keras model at scale on GCP" in the learning objectives</li><li>Remove "Once we have trained our model, we will see how to train our model at scale on GCP using AI Platform"</li><li>Add maybe a link to the original paper [Wide &amp; Deep Learning for Recommender Systems](<a href="https://arxiv.org/abs/1606.07792" target="_blank">https://arxiv.org/abs/1606.07792</a>)</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/576/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
these all look great. Thanks Benoit!!